error-handler.js

```diff
@@ -98,15 +99,16 @@ var mixIn = require('mout/object/mixIn'),
     stream.pipe(res);
   },
 
-  send = function send(statusCode, err, res, o) {
+  send = function send(statusCode, err, res, o) {    
+    if (!err.message) err.message = statusCodes[statusCode];
```

Comment:
Oops I documented a bit in the longer change description, basically ran into problems with mout, mixin call you suggested did not copy name from http-errors error object because that is non enumerable, so I made error the target and changed to fillin, but then ran into problem as fillin will not overwrite empty message string on error object as fillin does a null check, not a falsy check.